Makefile

```diff
@@ -21,80 +21,80 @@ BUILDDIR      = docs/build
 
 # Put it first so that "make" without argument is like "make help".
 help:
-	@echo "Please use 'make <target>' where <target> is one of"
-	@echo "  install                     install dependencies and prepare environment"
-	@echo "  install-dev                 install dependencies and everything needed to run tests"
-	@echo "  build-requirements          install all requirements and freeze them in requirements.txt"
-	@echo "  serve                       start the webpush_channels server on default port"
-	@echo "  migrate                     run the webpush_channels migrations"
-	@echo "  tests-once                  only run the tests once with the default python interpreter"
-	@echo "  flake8                      run the flake8 linter"
-	@echo "  tests                       run all the tests with all the supported python interpreters (same as travis)"
-	@echo "  clean                       remove *.pyc files and __pycache__ directory"
-	@echo "  distclean                   remove *.egg-info files and *.egg, build and dist directories"
-	@echo "  maintainer-clean            remove the .tox and the .venv directories"
-	@echo "  docs                        build the docs"
-	@echo "Check the Makefile to know exactly what each target is doing."
+    @echo "Please use 'make <target>' where <target> is one of"
```

Comment:
Actually we should not change tabs with space in the Makefile because the makefile only works with tabs. Did you try it?